_docs/setup/kubernetes/sidecar-injection.md

```diff
@@ -162,6 +162,57 @@ minikube start \
 	--extra-config=apiserver.Admission.PluginNames=NamespaceLifecycle,LimitRanger,ServiceAccount,PersistentVolumeLabel,DefaultStorageClass,DefaultTolerationSeconds,MutatingAdmissionWebhook,ValidatingAdmissionWebhook,ResourceQuota \
 	--kubernetes-version=v1.9.0
 ```
+
+### AWS (with Kops)
+
+When you install a new cluster with Kubernetes version 1.9, prerequisite for `admissionregistration.k8s.io/v1beta1` enabled is covered. 
+
+Nevertheless admission controllers needs to be updated.
```

Comment:
I think this should read "the list of admission controllers needs to be updated"?